slack_sdk/oauth/installation_store/models/installation.py

```diff
@@ -8,17 +8,23 @@
 class Installation:
     app_id: Optional[str]
     enterprise_id: Optional[str]
+    enterprise_name: Optional[str]
     team_id: Optional[str]
-    bot_token: str
```

Comment:
In javascript, I have these bot values set to optional. This might be because `@slack/oauth` in js supports classic slack apps, where in python we only seem to support `oauth.v2.access`.

Reply:
as you changed, we should make it optional

Reply:
I made `bot_token` optional in python. But are you saying I shouldn't make it optional?

Reply:
Ah no making it optional is the way to go. I just left a comment as I didn’t react here.